DNN Platform/Website/DotNetNuke.log4net.config

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I think this file got committed by mistake right?

Reply:
Yes, this was not intentional.